src/components/EditorTasks/EditorTasks.js

```diff
@@ -68,6 +69,27 @@ export const EditorTasks = () => {
     })
   }, [dispatch, markTasksId, tasks]);
 
+  const handleSplitTask = () => {
+    markTasksId.forEach(item => {
+
+      if (tasks[item].subtasks) {
+        Object.values(tasks[item].subtasks).forEach((subtask) => {
+          const restoredTask = {...subtask};
+
+          restoredTask.time = tasks[item].time;
+          restoredTask.status = tasks[item].status;
+          restoredTask.timeChange = tasks[item].timeChange;
+
```

Comment:
кажется лучше делать таким образом 
```suggestion
const restoredTask = {
...subtask,
time: tasks[item].time,
status: tasks[item].status,
timeChange: tasks[item].timeChange,
};
```

или же просто забирать весь task[items] через spread (но в случае если там нет ничего лишнего)

Reply:
до, согласен, так лучше.
Спасибо